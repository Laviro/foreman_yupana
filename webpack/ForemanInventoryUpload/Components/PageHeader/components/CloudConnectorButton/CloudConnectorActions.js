import { post } from 'foremanReact/redux/API';
import { CONFIGURE_CLOUD_CONNECTOR } from './CloudConnectorConstants';
import { inventoryUrl } from '../../../../ForemanInventoryHelpers';

export const configureCloudConnector = () =>
  post({
    key: CONFIGURE_CLOUD_CONNECTOR,
    url: inventoryUrl('enable_cloud_connector'),
  });
