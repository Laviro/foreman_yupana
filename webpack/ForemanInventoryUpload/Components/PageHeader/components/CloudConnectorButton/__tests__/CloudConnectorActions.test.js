import { testActionSnapshotWithFixtures } from '@theforeman/test';
import { configureCloudConnector } from '../CloudConnectorActions';

const fixtures = {
  'should configureCloudConnector': () => configureCloudConnector(),
};

describe('CloudConnector button actions', () =>
  testActionSnapshotWithFixtures(fixtures));
