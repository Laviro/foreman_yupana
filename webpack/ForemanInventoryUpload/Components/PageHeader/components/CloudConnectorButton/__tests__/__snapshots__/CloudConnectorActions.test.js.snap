// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloudConnector button actions should configureCloudConnector 1`] = `
Object {
  "key": "CONFIGURE_CLOUD_CONNECTOR",
  "type": "post-some-type",
  "url": "/foreman_inventory_upload/enable_cloud_connector",
}
`;
